test/pull-changed/expected-stdout

```diff
@@ -1,6 +1,6 @@
 Plan for "pull" operation:
   * C bar/extractor/ex-generic-v2/empty | changed: configuration
   * R bar/extractor/keboola.ex-db-mysql/with-rows/rows/disabled | changed: configuration, isDisabled
-  * C bar/transformation/keboola.python-transformation-v2/python-transformation | changed: configuration, description
+  * C bar/transformation/keboola.python-transformation-v2/python-transformation | changed: blocks, description
```

Comment:
Vo funkcionalnych testoch, iba tato drobna zmena, inak sa to sprava rovnako.
Ale `Config.Blocks` su definovane pre remote aj local stave -> daju sa na tom stavat dalsie veci.